AwsCryptographicMaterialProviders/runtimes/go/ImplementationFromDafny-go/SynchronizedLocalCMC/externs.go

```diff
@@ -0,0 +1,14 @@
+package SynchronizedLocalCMC
```

Comment:
TODO: Skeleton Implementaion / not tested. Blocked on Dafny Bug: https://t.corp.amazon.com/P157792076